CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.14.0 Release
+- Release Date: November 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
+
+**Combatibility Note: Mobile App v1.13+ is required for Mattermost Server v5.4+**
+
+### Bug Fixes
+- Fixed an issue where the Android app did not allow establishing a network connection with any server that used a self-signed certificate that had the CA certificate installed on the device.
```

Comment:
```suggestion
- Fixed an issue where the Android app did not allow establishing a network connection with any server that used a self-signed certificate that had the CA certificate user installed on the device.
```

Reply:
Not perfect but I think you need to emphasize that the issue was with user installed CAs not being used.